src/main/java/model/ExcelImporter.java

```diff
@@ -0,0 +1,70 @@
+package main.java.model;
+
+import java.io.FileInputStream;
+import java.util.ArrayList;
+
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.DataFormatter;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.ss.usermodel.WorkbookFactory;
+
+public class ExcelImporter {
+	
+	private Workbook wb;
+	
+	public ExcelImporter(String file) {
+		try {
+			this.wb = WorkbookFactory.create(new FileInputStream(file));
+		} catch(Exception ioe) {
+			System.out.println("Ficheiro n�o encontrado");
+		    ioe.printStackTrace();
+		}
+	}
+	
+	public ExcelRow getRow(int rowIndex) {
+		try {
+			
+			ArrayList<String> rowContent = new ArrayList<String>();
+			
+			Sheet sheet = wb.getSheetAt(0);
+			Row row = sheet.getRow(rowIndex);
+	       	
+			for (Cell cell : row) {
+	       		rowContent.add(new DataFormatter().formatCellValue(cell));
+	        }
+			
+			return new ExcelRow(rowContent);
+		} catch(Exception e) {
+			
+			System.out.println("Invalid row...");
+			System.out.println("Showing next valid row...");
+			
+			return getRow(++rowIndex);

```

Comment:
What I'm trying to do here is to get the next row. If I used rowIndex++, it would get the same row as it wouldn't increment the variable before the call of the method. That's why I use ++rowIndex. To be honest I'm not sure if what I'm trying to do makes sense